src/checkout-buttons/strategies/apple-pay/apple-pay-button-initialize-options.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * A set of options that are required to initialize ApplePay in cart.
+ *
+ * When ApplePay is initialized, an ApplePay button will be inserted into the
+ * DOM. When a customer clicks on it, it will trigger Apple sheet.
+ */
+ export default interface ApplePayButtonInitializeOptions {
+    /**
+     * The ID of a container which the sign-in button should insert into.
+     */
+    container: string;
```

Comment:
Do we need this?

Reply:
I have removed them. Only leave `onError`, `onPaymentAuthorize`, `buttonClassName` now.